content/en/core/releases/4.4.0.md

```diff
@@ -0,0 +1,128 @@
+
+---
+title: "4.4.0 release notes"
+linkTitle: "4.4.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.4.0%22).
+
+## Upgrade notes
+
+The deprecated hardcoded `national_admin` role no longer behaves as a CouchDb admin. The role now behaves as an ordinary online user role, with no additional implicit permissions.
+Configurations using `national_admin` role can either switch to using a CouchDb admin for admin-only tasks and/or granting more permissions to `national_admin`.
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+#### CHT dialogs.
+
+CHT dialogs were updated to align with [Material guidelines](https://m3.material.io/components/dialogs/guidelines). This includes updating the visual design and text of dialogs to align with best practices.
+
+{{< figure src="../images/4.4.0-8391-cht-dialogs.png" class="left col-12 col-lg-12" >}}
+<br clear="all">
+
+As part of this work, some translation keys changed. We have provided translations for all of the languages officially supported by CHT (English, Spanish, French, Indonesian, Hindi, Nepali, Bambara, Swahili). If you have a custom language and have any of these features, you will need to translate these in Admin app > Display > Translations tab.
```

Comment:
I don't recommend translating via the admin app because if the app is then updated using cht-conf these changes will get overridden.

```suggestion
As part of this work, some translation keys changed. We have provided translations for all of the languages officially supported by CHT (English, Spanish, French, Indonesian, Hindi, Nepali, Bambara, Swahili). If you have a custom language and have any of these features, you will need to translate these in your configuration files.
```